platform/ATCmdParser.h

```diff
@@ -288,6 +288,16 @@ class ATCmdParser : private NonCopyable<ATCmdParser>
      * recv operation.
      */
     void abort();
+    
+    /**
+    * Process out-of-band data
+    *
+    * Process out-of-band data in the receive buffer. This function
+    * returns immediately if there is no data to process.
+    *
+    * @return true if oob data processed, false otherwise
+    */
```

Comment:
formatting, should match comment style in file

Reply:
@SenRamakri Any update or respond to this ?